src/elements/radio-button/radio-button-group/radio-button-group.spec.ts

```diff
@@ -71,13 +71,13 @@ import '../radio-button-panel.js';
         expect(element.value).to.be.equal(radio.value);
       });
 
-      it('should ignore disabled radios', async () => {
+      it('should respect disabled radio in value getter', async () => {
         const radio = radios[0];
         radio.checked = true;
         radio.disabled = true;
         await waitForLitRender(element);
 
-        expect(element.value).to.be.null;
+        expect(element.value).to.be.equal('Value one');
```

Comment:
Do you think this is the desired behavior? Natively, the FormData would be empty in this case

Reply:
Is it the same? Here it is the group while native there is no group?

Reply:
My doubt is, should the `group.value` consider disabled radios?

In this case, both the native and our implementation would return an empty FormData (as if nothing were selected).
Meanwhile the `sbb-radio-group` returns `Value one`. I feel it is inconsistent :/